DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -454,7 +472,9 @@ Function Upgrade-Package {
         [Parameter(Position=2)]
         [string]$pSource,
         [Parameter(Position=3)]
-        [string]$cParams
+        [string]$cParams,
+        [Parameter(Position=4)]
+        [string]$pVersion

```

Comment:
If you look at line 146 above you've got:

```
$chocoParams += " --version='$versionToInstall'"
Upgrade-Package -pName $Name -pParams $Params -pSource $Source -cParams $chocoParams
```

The version parameter is added in the calling code by adding it to `-cParams` parameter. So I'm unsure why we need a specific parameter here that does nothing more than add it into the Choco parameters?